net/libreswan/files/etc/hotplug.d/libreswan/02-vti

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
```

Comment:
Shouldn't we use the [proto vti](https://github.com/openwrt/openwrt/blob/master/package/network/config/vti/Makefile) here? This would integrate better into openwrt (firewall handling).

Reply:
Can you explain a bit more in detail?

do you mean to add the vti interface in UCI from there or, use the vti interface from UCI instead of creating here?

Reply:
At first I would use a already created vti interface first created by luci-proto-vti.
But would it not be better to use xfrm? But I am not sure if libreswan does support this.

Reply:
they added xfrm support, can't remember if I did had some issues with it

Reply:
Added support of xfrm and ipsec from uci.

The only limitation for libreswan would be, It expects xfrm interface name in the format ipsec<ifid>.